hyperlane/relayer_runner.go

```diff
@@ -50,18 +51,6 @@ func (r *RelayerRunner) Run(ctx context.Context) error {
 		case <-ctx.Done():
 			return nil
 		case <-ticker.C:
-			// find settlement txns in the db and insert them as pending
-			// hyperlane txns in the hyperlane table
-			if err := r.findSettlementsToRelay(ctx); err != nil {
-				return fmt.Errorf("finding settlements txns to relay: %w", err)
-			}
-
-			// find timeout txns in the db and insert them as pending hyperlane
-			// txns in the hyperlane table
-			if err := r.findTimeoutsToRelay(ctx); err != nil {
-				return fmt.Errorf("finding timeout txns to relay: %w", err)
-			}
-
```

Comment:
no longer polling the db for timeouts rows and settlement rows, these services will call the SubmitTxToRelay function on the relayer runner. this forces services that need hyperlane relaying to explicitly require the relayer runner as a dependency